packages/shared/lib/services/notification/slack.service.ts

```diff
@@ -532,7 +532,7 @@ export class SlackService {
         provider: string;
     }): Promise<void> {
         const update = await db.knex.transaction(async (trx) => {
-            const slackNotificationsEnabled = await environmentService.getSlackNotificationsEnabled(nangoConnection.environment_id);
+            const slackNotificationsEnabled = await environmentService.getSlackNotificationsEnabled(nangoConnection.environment_id, trx);
```

Comment:
this one is problematic, it is on a very frequently executed code patch (ie: every sync or action successful completion)